src/index.html

```diff
@@ -11,6 +11,26 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="http://localhost:8080/api" 
+      method="post"
+      class="search-bar big">
```

Comment:
fix closing tag + let's a big bar would be default, add all styles for big form to search-bar, then rewrite some of them for smaller form with small class

```suggestion
    <form 
      action="http://localhost:8080/api" 
      method="post"
      class="search-bar"
    >
```

Reply:
fix everywhere